build-logic/jvm/src/main/kotlin/gradlebuild.unittest-and-compile.gradle.kts

```diff
@@ -224,6 +224,10 @@ fun Test.configureJvmForTest() {
             jvmArgs(listOf("--add-opens", "java.base/java.lang=ALL-UNNAMED")) // Used in tests by ClassLoaderUtils
         }
     }
+    if (usesEmbeddedExecuter() && OperatingSystem.current().isWindows) {
+        // Disable incremental snapshotting for Windows, since it throws OOM
```

Comment:
The comment should link to the root cause issue at https://youtrack.jetbrains.com/issue/KT-57757/Reduce-classpath-snapshotter-memory-consumption